src/test/java/uk/gov/hmcts/reform/em/stitching/rest/errors/ExceptionTranslatorTestController.java

```diff
@@ -83,4 +84,9 @@ public void setTest(String test) {
     public static class TestResponseStatusException extends RuntimeException {
     }
 
+    @GetMapping("/test/no-such-element")
+    public void handleNoSuchElementException() {
+        throw new NoSuchElementException("No such Element!");
```

Comment:
If this error message gets passed into CCD, could you pass the element name into the message, to make the error message more informative?